server/src/main/scala/za/co/absa/atum/server/model/SuccessResponse.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.model
+
+object SuccessResponse {
```

Comment:
What about `ErrorResponse` ? I guess that since you haven't implemented yet, this would be a part of some future ticket?

Also, some metadata on the base envelope level (that even `SuccessResponse` and `ErrorResponse` would share), such as `requestID` and `requestURI` ? Maybe at least the `requestID` could be addressed here in this ticket? I.e. server-side generation of these

Reply:
`ErrorResponse` type already exists.

Reply:
My apologies, I overlooked it - okay, but there is no hierarchy between them, i.e. no common ancestor that very likely we'll have when we start adding the metadata into the envelope

Reply:
so basically by adding it and `requestId` there, I think that it's a good step forward that we can easily extend later

Reply:
Update: okay, understood, it seems that the only thing `could` be the `requestID` - let's check with the team then, I feel that we could push it forward quite easily hopefully